.buildkite/scripts/cloud-cleanup.sh

```diff
@@ -25,26 +25,43 @@ export DELETE_RESOURCES_BEFORE_DATE
 
 CLOUD_REAPER_IMAGE="${DOCKER_REGISTRY}/observability-ci/cloud-reaper:0.3.0"
 
-DRY_RUN="$(buildkite-agent meta-data get DRY_RUN --default "${DRY_RUN:-"true"}")"
+DRY_RUN="$(buildkite-agent meta-data get DRY_RUN_DEPRECATED --default "${DRY_RUN:-"true"}")"
 
 resources_to_delete=0
 
 COMMAND="validate"
+redshift_message=""
 if [[ "${DRY_RUN}" != "true" ]]; then
     # TODO: to be changed to "destroy --confirm" once it can be tested
     # that filters work as expected
     COMMAND="plan"
+    redshift_message=" - stale redshift clusters will be deleted"
 else
     COMMAND="plan"
 fi
 
+buildkite-agent annotate \
+  "[${BUILDKITE_STEP_KEY}] Running DRY_RUN (${DRY_RUN}) using cloud-reaper command \"${COMMAND}\"${redshift_message}" \
+  --context "ctx-cloud-reaper-info-deprecated" \
+  --style "info"
+
 any_resources_to_delete() {
```

Comment:
Are these methods duplicated? Could we move them to some shared file?

Reply:
This file is going to be deprecated by the new file introduced in #2567 once integrations also use the new system cc @v1v.

So I thought it's not needed to do refactors fow now here.